app/forms/declaration/TransportPayment.scala

```diff
@@ -36,14 +38,14 @@ object TransportPayment extends DeclarationPage {
   val eFunds = "H"
   val accHolder = "Y"
   val notPrePaid = "Z"
+  val notAvailable = "_"
```

Comment:
`_` might cause problems when used in html forms. Have checked it is working and correct value is stored in DB?

Reply:
Its a standard ASCII char so I don't see an issue. I just didn't want to use a A->Z character in case in future they decide to add another option with that value

Reply:
AC's have been updated to store the '_' value and they all pass: https://github.com/hmrc/cds-exports-acceptance/pull/698/files#diff-d489df75c5973ab0eadec1a8b8c52e66a37ab7a21f6f24432f0ce9595fd84ed5